### PR TITLE
delete Train Schedule V1 endpoints

### LINKED ISSUE #8263: DROP TSV1: migrate tables

### LINKED ISSUE #8264: DROP TSV1: delete endpoints editoast

### PR DESCRIPTION
**Pull request by younesschrifi**
Close #8264 
Close #8263

### CODE REVIEW

File: tests/fuzzer/fuzzer_v2.py [hunk 1]
```diff
@@ -228,6 +228,7 @@ def _test_new_train(
     """
     print("testing new train")
     schedule_payload = _make_payload_schedule(path, rolling_stock)
+    print(schedule_payload)
```
Comment (shenriotpro) [inline tests/fuzzer/fuzzer_v2.py]: do we need this?

File: tests/tests/test_stdcm.py [hunk 2]
```diff
@@ -133,6 +127,7 @@ def test_work_schedules(small_scenario_v2: Scenario, fast_rolling_stock: int):
         },
     )
     work_schedules_response = work_schedules_r.json()
+    print(work_schedules_response)
```
Comment (shenriotpro) [inline tests/tests/test_stdcm.py]: do we need this?

File: tests/tests/test_train_schedule.py [hunk 3]
```diff
@@ -1,81 +1,68 @@
...
-        json={"path_id": None, "train_ids": west_to_south_east_simulations[0]},
+        raise RuntimeError(f"Schedule error {r.status_code}: {r.content}, payload={json.dumps(trains_ids)}")
+    r = requests.get(
+        f"{EDITOAST_URL}v2/train_schedule/{trains_ids[0]}/",
     )
-    assert r.status_code == 422
-    r = requests.post(
-        f"{EDITOAST_URL}train_schedule/results/",
-        json={"path_id": None, "train_ids": west_to_south_east_simulations[1]},
+    print(r.content)
```
Comment (shenriotpro) [inline tests/tests/test_train_schedule.py]: do we need this?

**Review by shenriotpro — APPROVED** [review]
Thanks, LGTM for tests/

File: front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx
Comment (clarani) [inline front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx:1]: You forgot a comment l62

File: front/src/utils/types.ts [hunk 4]
```diff
@@ -21,3 +21,9 @@ export const mapBy = <K extends keyof T, T>(items: T[] | undefined, key: K): Map
 
 export const concatMap = <K, V>(map1: Map<K, V>, map2: Map<K, V>) =>
   new Map([...map1.entries(), ...map2.entries()]);
+
+export type RangedValue = {
+  begin: number;
+  end: number;
+  value: string;
+};
```
Comment (clarani) [inline front/src/utils/types.ts]: Can you move this in common/types instead please ? :)

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark: 

Feel free to close my comments

File: editoast/src/models/mod.rs
Comment (flomonster) [inline editoast/src/models/mod.rs:1]: Do you plan to delete the `models` module completely? We shouldn't use models v1 anymore.

**Review by flomonster — APPROVED** [review]
LGTM. 🗑️ ♻️ 🧹

File: editoast/src/models/mod.rs
Comment (leovalais) [reply editoast/src/models/mod.rs:1]: You can also delete the `Model` macro in `editoast_derive`. The `s/ModelV2/Model` is probably for a follow up PR though.

Comment (younesschrifi) [reply editoast/src/models/mod.rs:1]: I will do it in a different PR as it is an Enhancement (https://github.com/OpenRailAssociation/osrd/issues/5056).

**Pull request merged**